araproc/framework/config_files/analysis_configs.yaml

```diff
@@ -14,6 +14,17 @@ station100:
       filt3 : { min_freq : 0.396, max_freq : 0.410, min_power_ratio : 0.02 }
       filt4 : { min_freq : 0.570, max_freq : 0.580, min_power_ratio : 0.02 }
 
+  readout_limits:
```

Comment:
I think these are config dependent? So they probably need to be indented and put into each configuration?